package.json

```diff
@@ -31,5 +32,8 @@
     "generate-types": "typechain --target=ethers-v5 'client/src/contracts/*.json' --out-dir client/src/contracts",
     "migrate": "tsc -p ./tsconfig.migrate.json --outDir ./migrations && truffle migrate",
     "typecheck": "truffle compile && pnpm generate-types && tsc --noEmit"
+  },
+  "devDependencies": {
+    "@types/react-paginate": "^7.1.1"
```

Comment:
can you explain why this is dev dependencies

Reply:
When I tried to install react-paginate, there was some error. So, I tried to resolve that error. Then this got created.